tools/migrate_photo_to_odp.rb

```diff
@@ -4,20 +4,11 @@
 batch_size = 100000
 index = 0
 
-static_bucket_id = FilePrefix.where( prefix: "https://static.inaturalist.org/photos" ).first.id
-
 while index <= max_photo_id
   puts "Starting batch #{index}"
-  LocalPhoto.joins( "LEFT JOIN flags ON (photos.id = flags.flaggable_id)" ).
-  where( "flags.id IS NULL" ).
-  where( "photos.file_prefix_id=?", static_bucket_id ).
-  where( "photos.license not in (?)", Shared::LicenseModule::LICENSE_NUMBERS - Shared::LicenseModule::ODP_LICENSES).
-  where( "photos.id between ? and ?", index, index + batch_size - 1 ).
-  each do |photo|
-    LocalPhoto.delay(
+  LocalPhoto.delay(
       queue: "photos",
-      unique_hash: { "LocalPhoto::change_photo_bucket_if_needed": photo.id }
-    ).change_photo_bucket_if_needed( photo )
-  end
+      unique_hash: { "LocalPhoto::migrate_to_odp_bucket": index }
```

Comment:
we can add priorities to delayed jobs. Since we know these will take a while and we don't need them to happen immediately, maybe we should add `priority: INTEGRITY_PRIORITY` so if other photo moving jobs come in from user actions they could be run before all of these longer running jobs finish